### PR TITLE
Fix the definition of WT_STREAM

### PR DESCRIPTION
**Pull request by martinthomson**
The definition was a little loose, so I rewrote it.

We only need to use 0x0a and 0x0b, which is lucky and it lets us avoid
much of the mess that QUIC has around bits in the type.

I've also added definitions for the fields in the frame.

### CODE REVIEW

File: draft-ietf-webtrans-http2.md [hunk 1]
```diff
@@ -376,16 +376,13 @@ The WT_STOP_SENDING frame defines the following fields:
 
 ## WT_STREAM Frames
```
[the remark this answers is not in the document]
Comment (ekinnear) [reply draft-ietf-webtrans-http2.md:377]: Started that way, but opted to follow what 9000 did and keep them plural

File: draft-ietf-webtrans-http2.md [hunk 2]
```diff
@@ -396,6 +393,16 @@ WT_STREAM Frame {
...
 
+WT_STREAM frames contain the following fields:
+
+Stream ID:
+: The stream ID for the stream.
+
+Stream Data:
+: Zero or more bytes of data for the stream.  Empty WT_STREAM frames MUST NOT be
+used unless they open or close a stream; an endpoint MAY treat an empty
+WT_STREAM frame that neither starts nor ends a stream as a session error.
```
Comment (ekinnear) [inline draft-ietf-webtrans-http2.md:404]: Ah, nice. We definitely need the ability to open/close without sending, but no need to leave that wide open.

File: draft-ietf-webtrans-http2.md [hunk 3]
```diff
@@ -376,16 +376,13 @@ The WT_STOP_SENDING frame defines the following fields:
 
 ## WT_STREAM Frames
 
-WT_STREAM frames implicitly create a stream and carry stream data. The Type
-field in the WT_STREAM frame takes the form 0b00001XXX (or the set of values
-from 0x08 to 0x0f) to maximize compatibility with QUIC. However, unlike QUIC,
-there are only one bit used to determine the fields that are present in the
-frame:
+WT_STREAM frames carry stream data.
```
Comment (ekinnear) [inline draft-ietf-webtrans-http2.md:379]: Asides from whether or not this really needs its own paragraph, do we somewhere want to keep that a stream frame creates a stream? (no preference around wording/implicitness/that stuff, but we no longer say in this section at all that it creates a stream)

Comment (martinthomson) [reply draft-ietf-webtrans-http2.md:379]: The definition starts with "WT_STREAM frames implicitly create a stream [...]"  I figured that would be OK.

Comment (ekinnear) [reply draft-ietf-webtrans-http2.md:379]: Sorry, I must just be misreading the diff. It looks to me like 
> implicitly create a stream

is no longer in the document anywhere, which is fine from a wording perspective, but we probably want _some_ mention that they create a stream.

Comment (ekinnear) [reply draft-ietf-webtrans-http2.md]: Put create a stream back in that sentence in [60b744e](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/45/commits/60b744e1b8e925f2a87550d895192c4d8023190f), but we can always reword/take it out later if needed.

**Pull request merged**